src/TheAlgorithms.jl

```diff
@@ -70,8 +70,15 @@ export sum_gp
 export surfarea_cube
 export surfarea_sphere
 export trapazoidal_area
-export eratosthenes
```

Comment:
I am confused, while in my fork, the eratosthenes algorithms is synchronous with this repo, but the pull request displays that I delete the algorithm export and algorithm include in **TheAlgorithms.jl** file

Reply:
I moved it above to have it in alphabetical order. So it should not be missing